### ISSUE TITLE
Report when table cells extend outside border in Word document

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#10806]
### Is your feature request related to a problem? Please describe.
When working on a table in Word, it is possible to resize the table such that it extends past the border on a page.  NVDA provides no indication that this has occurred either when it is done or when navigating the table.

Steps to reproduce:
1) Open Word :)
2) Insert a three column table - press ALT+N, T, then right arrow twice and down arrow to select a three column, two row table, and press ENTER.
3) The focus starts in the first cell.  Press SHIFT+RIGHT ARROW three times to select the top cell in each column.
4) Open the context menu and press R to open "Table Properties"
5) Press alt+u to select the "Column" page.
6) Press TAB until the focus is in the width spin edit.
7) Type a value which is say half the total width of the page (eg 10 centimeters if using metric measurements and A4 size paper).
8) Press ENTER to OK the change to the three columns and resize the table.
9) Use the right arrow to move to column 3, then press NVDA+numpad delete (laptop: NVDA+delete) to report the current location.  NVDA reports something like 23cm.  This is well past the right edge of the page, however there is no indication of this.

### Describe the solution you'd like

When navigating into a table or to a cell which goes outside the page margin, NVDA should report this.

### Additional context
Using NVDA 2019.3.1 and Office 365 v 16.0.12430.20198  although I think this has always behaved the same.

**Comment by Brian1Gaff** [comment]
Yes it has, and its made some printouts look decidedly, ahem, interesting. 
grin.

Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
